Extension/package.nls.json

```diff
@@ -400,5 +400,27 @@
     "c_cpp.debuggers.logging.category.verbose.description": "Logs that are used for interactive investigation during development. These logs should primarily contain information useful for debugging and have no long-term value.",
     "c_cpp.debuggers.logging.category.warning.description": "Logs that highlight an abnormal or unexpected event in the application flow, but do not otherwise cause the application execution to stop.",
     "c_cpp.debuggers.logging.category.error.description": "Logs that highlight when the current flow of execution is stopped due to a failure. These should indicate a failure in the current activity, not an application-wide failure.",
-    "c_cpp.debuggers.logging.category.none.description": "Not used for writing log messages. Specifies that a logging category should not write any messages."
+    "c_cpp.debuggers.logging.category.none.description": "Not used for writing log messages. Specifies that a logging category should not write any messages.",
+    "c_cpp.walkthrough.title": "Get Started with C++ Development",
+    "c_cpp.walkthrough.description": "Dive into VS Code's rich C++ development experience.",
+    "c_cpp.walkthrough.set.up.title": "Set up your C++ Environment",
+    "c_cpp.walkthrough.activating.description": "Activating the C++ Extension to determine whether your C++ Environment has been set up.\nActivating Extension...",
+    "c_cpp.walkthrough.no.compilers.description": "We could not find a C++ compiler on your machine, which is required to use the C++ Extension. Follow the instructions on the right to install one, then click “Find my new Compiler” below.\n[Find my new Compiler](command:C_Cpp.RescanCompilers)",
```

Comment:
"Find my new Compiler" casing -- should be title case or sentence case

Reply:
Maybe these should be setence case?

Reply:
I see other walkthrough buttons use sentence casing

Reply:
actually, I see title case too

Reply:
Should be title case - but because new is less than 4 letters it should not be capitalized, only Find and Compiler

Reply:
So it's correct? News to me.